bokehjs/src/lib/models/widgets/sliders/categorical_slider.ts

```diff
@@ -57,16 +72,12 @@ export class CategoricalSlider extends AbstractSlider<string> {
   declare properties: CategoricalSlider.Props
   declare __view_type__: CategoricalSliderView
 
-  declare value: string
-  declare value_throttled: string
-
```

Comment:
These changes need an explanation and most likely reversal. Mind that `value*` properties are of type `unknown` in the base class and we would like to have a more specific type (`string`) here.